draft-regext-rdap-extensions.md

```diff
@@ -254,13 +279,21 @@ path that is not defined by that RDAP extension, those query parameter
 names SHOULD be constructed in the same manner as URL path segments
 (that is, extension identifier + '_' + parameter name).  (See section
 (#identifier_omission) regarding when an extension identifier may be
-omitted.)
-
-When an RDAP extension defines query parameters to be used with a URL
-path defined by that RDAP extension, prefixing of query parameters is
-not required.  In this situation, the URL path operates as a namespace
-for the query parameters, so there is no risk of collision with
-parameters defined elsewhere.
+omitted and Section #(bare_extension) regarding bare extensions.)
+
+Though the URL path operates as a namespace, both [@!RFC8982]
```

Comment:
The text that has been removed here looks like it should still be present between this paragraph and the previous one.